### PR TITLE
Add Rating Component

### PR DESCRIPTION
**Pull request by varun13169**
Add Rating Component

 - Ran Beautify on Other components

### CODE REVIEW

**Comment by varun13169** [comment]
@zoef-yt

I have addressed the comment mentioned in the following closed PR: https://github.com/varun13169/delta-ui/pull/27

File: library/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.dui-rating__step:hover, .dui-rating__step--filled {
+    stroke: var(--primary-color);
+    fill: var(--primary-color);
+}
+
+/* Heart */
+.dui-rating__step-heart {
+}
```
Comment (zoef-yt) [inline library/rating/rating.css:18]: for future reference or accidentally left?

**Review by zoef-yt — APPROVED** [review]
Good and clean code. Good to merge.

File: library/rating/rating.css (hunk 1 above)
Comment (varun13169) [reply library/rating/rating.css:18]: Future Ref.

File: library/rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    stroke: var(--primary-color-p2);
+}
+
+.dui-rating__step:hover, .dui-rating__step--filled {
+    stroke: var(--primary-color);
+    fill: var(--primary-color);
+}
+
+/* Heart */
+.dui-rating__step-heart {
```
Comment (archit7448) [inline library/rating/rating.css:17]: why this is empty

**Review by archit7448 — APPROVED** [review]
Good to merge

File: library/rating/rating.css (hunk 2 above)
Comment (varun13169) [reply library/rating/rating.css:17]: This is a placeholder for future use.

**Pull request merged**